metadata_annotation.ipynb

```diff
@@ -0,0 +1,2307 @@
+{
```

Comment:
<blockquote>This can help augment existing datasets with more datasets to increase sample sizes for training of deep learning models and add prior knowledge of data that cannot be seen initially. </blockquote><p>Not just DL, but generally ML. We're not using DL in this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>